src/3box.js

```diff
@@ -202,13 +202,13 @@ class Box {
       ipfs = globalIPFS
       usingGlobalIPFS = true
     } else {
-      ipfs = await initIPFS(opts.ipfs, opts.iframeStore)
+      ipfs = await initIPFS(opts.ipfs, opts.iframeStore, opts.ipfsOptions)
```

Comment:
`iframeStore` will be undefined, not sure if that can generate an error

Reply:
nope, that's fine (see the `initIPFS` code)